usr/share/rear/lib/nbkdc-functions.sh

```diff
@@ -0,0 +1,54 @@
+# locate NovaBACKUP DC client configuration, referenced by
+#
+# prep/NBKDC/default/40_prep_nbkdc.sh
+# skel/NBKDC/etc/scripts/system-setup.d/59-start-nbkdc-client.sh
+
+
+# Set the NovaBACKUP DC default directory
+NBKDC_DIR=/opt/NovaStor/DataCenter
```

Comment:
In that case I suggest to restructure your code so that during mkrescue it will do all the discovery and that in the rescue system everything is already set. Simply generate the required scripts in the rescue system. It can also be a good idea to generate some kind of a `nbkdc_settings.sh` somewhere that contains all the paths and other stuff that you found out during mkrescue. Then all other scripts can simply source that and you are done. The result is a single point of configuration that is much easier to debug than having autodiscovery everwhere.